### ISSUE TITLE
'Display real name'/'Display as Anonymous' should be a functional link

### ISSUE THREAD

**Issue by draco-malfoy** [topcoder-archive/TCO21-Regionals-QA-Competition#58]
**Target URL**
https://promotions.newegg.com/international/global/index.html

**Steps to Reproduce**
1. go to https://www.newegg.com/global/in-en/
2. create account and login
3. go to profile> Account settings
4. click on 'Display real name'/'Display as Anonymous'

**Actual Result(s)**
'Display real name'/'Display as Anonymous' is displayed as static text. 
To make it work user has to click on edit and then choose 'Display real name'/'Display as Anonymous'.

**Expected Result(s)**
When clicked it should toggle between 'Display real name'/'Display as Anonymous' .

If clickable is not required, then is should show **Displaying real name** or **Displaying anonymous name** while displaying real name or anonymous name respectively

**Testing collateral:**

https://user-images.githubusercontent.com/46344490/128194688-66f8ea86-c248-47b6-93e4-25b589453366.mp4

![7](https://user-images.githubusercontent.com/46344490/128194848-9a4da7a7-91ec-4c85-b950-cd5d5eb3a037.JPG)

**Environment Details**
Device(s): Asus VivoBook
Resolution: 1536 x 864
Operating System: Windows 10 Home
Browser(s): Version 92.0.4515.107 (Official Build) (64-bit)

**Comment by macs054** [comment]
Not a bug. Like you presented in the video you can change this in the Edit section. It's purpose is just to show if your name is displayed as Anonymous or not, for what purpose I'm not sure, maybe in reviews, etc.


https://user-images.githubusercontent.com/1199534/128249251-83d063db-62dd-4973-a339-9619fa99190c.mp4

**Comment by draco-malfoy** [comment]
@macs054 agree, but there is also other condition I have written "If clickable is not required, then is should show Displaying real name or Displaying anonymous name while displaying real name or anonymous name respectively".  

that is, if the links are not clickable, then they should shown **'Displaying as X**' rather than '**Display as X'**

cc: @codejamtc

**Comment by codejamtc** [comment]
> When clicked it should toggle between 'Display real name'/'Display as Anonymous' .
- It is just an indicator to see what is your current selected option.  It's not a functional bug; 

> If clickable is not required, then is should show Displaying real name or Displaying anonymous name while displaying real name or anonymous name respectively
- This is a UX suggestion

**Submitter**: 0 Points
**Challenger**: 8 Points
